demos/python_demos/multi_camera_multi_person_tracking/requirements.txt

```diff
@@ -1,3 +1,18 @@
-opencv-python
-numpy
-scipy
+Cython>=0.29.12
+setuptools
+gdown>=3.8.3
+h5py>=2.9.0
+lxml>=4.3.4
+matplotlib>=3.0.3
+motmetrics>=1.1.3
+pandas>=0.24.2
+opencv-python>=3.4.5.20
+Pillow>=6.1.0
+pycocotools>=2.0.0
+requests>=2.22.0
+numpy>=1.16.4
+scipy>=1.3.0
+six>=1.12.0
+tensorboard>=1.8
+tqdm>=4.32.2
+yacs>=0.1.6
```

Comment:
This file should only list direct dependencies, so as to make it clear what the demo actually needs. Could you remove the packages that are not used directly?

Reply:
Fixed.

Reply:
It doesn't seem to actually be fixes. E.g. I can immediately see `requests`, which is not used anywhere in the demo.

Reply:
My bad. Fixed.